### DISCUSSION TITLE
is there a way to edit custom parse the html before prompting the llm.

### DISCUSSION THREAD

**Discussion by goodenergi** [ScrapeGraphAI/Scrapegraph-ai#439] [Q&A]
the context legth of the html is over doule what the llm allows. but im only using scrapegraph ai to generate scraping script. so as long as structure is preserved i can remove alot of the repetetive "content", and still get the script working. 

dont know if this is a good idea.

**Comment by VinciGit00** [comment]
download the code and pass it later
